j5/boards/base.py

```diff
@@ -78,8 +78,8 @@ def make_all_safe() -> None:
 class BoardGroup:
     """A group of boards that can be accessed."""
 
-    def __init__(self, board: Board, backend: Backend):
-        self.board_class: Board = board
+    def __init__(self, board: Type[Board], backend: Backend):
```

Comment:
While we're at it, I think `backend` is supposed to be a `Type[Backend]` here, because we later call `backend.discover()` (or will do after #195 is merged). Also, there isn't a board in this context that `backend` would correspond to if it was an instance rather than a subclass.

Reply:
This should be fixed in #195.

Reply:
I have made #198 to track this.